.github/workflows/wasm.yml

```
@@ -7,10 +7,28 @@ on:
     branches: [master]
 
 jobs:
-  carton_wasmer_test:
+  carton_wasmer_test_5_3:
+    name: "Execute tests on WASM (Swift 5.3)"
```

nit: [according to the WebAssembly spec](https://webassembly.github.io/spec/core/intro/introduction.html#id1), correct capitalization is WASM
```suggestion
    name: "Execute tests on Wasm (Swift 5.3)"
```